config/kubernetes/kustomization.yaml

```diff
@@ -2,6 +2,9 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
 - mutatingwebhookconfiguration.yaml
-bases:
-  - ../common
-  - ../crd/default
+- ../common
+- ../crd/default
+images:
+- name: controller
+  newName: quay.io/dynatrace/dynatrace-operator
+  newTag: snapshot-refactor-remove-related-image-env
```

Comment:
On purpose?

Reply:
generated by the makefile. I don't think there is any harm in committing it, since it will be regenerated every time on "make bundle" anyways